asdf/_display.py

```diff
@@ -114,6 +125,15 @@ def __init__(
         self.recursive = recursive
         self.visible = visible
         self.children = []
+        self.tag = tag
+
+    def supports_info(node):
```

Comment:
This is being called as a class method up above, but it's not decorated as such and doesn't have a `cls` argument.  On the other hand, it seems to be working...